src/buildercore/fastly/vcl/office-webdav-200.vcl

```diff
@@ -0,0 +1,11 @@
+if(
+    req.http.User-Agent ~ "Microsoft Office Protocol Discovery"
+    ||
+    req.http.User-Agent ~ "Microsoft Office Existence Discovery"
+    ||
+    req.http.User-Agent ~ "Microsoft-WebDAV-MiniRedir"
+    ||
+    (req.http.User-Agent ~ "ms-office" && req.http.User-Agent !~ "Microsoft Outlook")
+) {
+    error 200;
```

Comment:
This should result in a 200 Varnish error page. Alternative is to use a `9xx` code and pick it up in `vcl_error`, but requires another VCL file.

Reply:
If 200 works (shouldn't be visible to an end user)